docker/README.md

```diff
@@ -1,22 +1,21 @@
-# GRASS GIS docker
-
-## GRASS GIS docker matrix
+## GRASS GIS 8.4 (current) Docker installation and usage
```

Comment:
Is there a way to not have to change this too when changing versions?

Reply:
Indeed, including a version anywhere (including the code examples below) slows the release process (here specifically the branching) adding time which at this point human needs to spend.